app/pages/Accounts/useAccountSync.ts

```diff
@@ -0,0 +1,75 @@
+import { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+    chosenAccountSelector,
+    chosenAccountInfoSelector,
+    updateAccountInfo,
+} from '~/features/AccountSlice';
+import {
+    updateTransactions,
+    loadTransactions,
+} from '~/features/TransactionSlice';
+import { noOp } from '~/utils/basicHelpers';
+import { AccountStatus } from '~/utils/types';
+import AbortController from '~/utils/AbortController';
+
+// milliseconds between updates of the accountInfo
+const accountInfoUpdateInterval = 30000;
+
+export default function useAccountSync() {
```

Comment:
Maybe add some documentation and explain that this keeps the chosen account's accountInfo and transactions in sync.